unittests/dojo_test_case.py

```diff
@@ -518,6 +521,11 @@ def reimport_scan_with_params(self, test_id, filename, scan_type='ZAP Scan', eng
         if test_title is not None:
             payload['test_title'] = test_title
 
+        if scan_date is not None:
+            payload['scan_date'] = scan_date
+        else:
+            payload['scan_date'] = str(timezone.localtime(timezone.now()).date())
```

Comment:
I think this should now be None as there no default anymore

Reply:
Yes good catch